package.json

```diff
@@ -11,7 +11,10 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "classnames": "^2.2.5",
+    "edx-bootstrap": "^0.2.1",
```

Comment:
I had to move all 3 of these into `dependencies` so that edx-platform would install them, but I'm not 100% sure I remember why.

Reply:
I think it had to do with the loaders I places in edx-platform's webpack config

Reply:
Resolved thanks to @ari - we ought to move these deps into edx-platform, since its webpack process is the one that needs them.

Reply:
xposted from HipChat, preserving here from posterity: I'd recommend keeping these as devDependencies in SFE, but adding them as dependencies in edx-platform. Since edx-platform essentially owns its own webpack process, it makes sense for it to have explicit dependencies for packages its webpack process consumes. Normally we'd add these as devDependencies in edx-platform as well, but due to the way we build and deploy assets, anything that's required for a production build needs to go in `dependencies`.